07LessonSeven/homework/css/style.css

```diff
@@ -0,0 +1,68 @@
+*{
+  font-family:sans-serif ;
+}
+nav{
+  background-color:#313131;
+  color:white;
+  height:80px;
+  margin-bottom:20px;
+}
+a{
+  color:white;
+  text-decoration:none;
+  float:left;
+  margin:30px 20px;
+}
+.first_link{
+  padding-left:80px;
+}
+h1{
+  padding-left:100px;
+}
+.green{
+  background-color:#55a747;
+  color:white;
+  font-size: 14px;
+  padding-top:50px;
+  padding-bottom:30px;
+  padding-left:30px;
+  padding-right:30px;
+  width:40%;
+  float:left;
+}
```

Comment:
Hi Nicole! You could do something like this:

+.common{
+  color:white;
+  font-size: 14px;
+  padding-top:50px;
+  padding-bottom:30px;
+  padding-left:30px;
+  padding-right:30px;
+  width:40%;
+  float:left;
+}

.green {
+  background-color:#55a747;
}

.gray {
+  background-color:gray;
}

<div class="common green">
....</div>

<div class="common gray">
.....</div>